_docs/setup/kubernetes/multicluster-install.md

```diff
@@ -52,65 +45,58 @@ has been validated with multicluster.
+context typically located in `$HOME/.kube/config`.
+
+## Gather credential files from remote
+
+For each remote cluster, execute the following steps:
 
-A unique security context is required to describe how to securely access
-each Kubernetes cluster in the system.  In the above example,
-`falkor07.kube.conf` is the Istio control plane credential file. The
-`config.istio.io/pilotCfgStore: True` flag is set in the condition. This
-`clusterregistry.k8s.io` descriptor describes the Istio control plane Kubernetes
-cluster.  For remotes, please set `pilotCfgStore: False`.
+1. Copy the credentials file on the remote to the Istio control
+plane cluster to `$HOME/multicluster/${CLUSTER_NAME}`.  The
+`${CLUSTER_NAME}` should be unique per remote.
 
-> The implementation only uses the
-`config.istio.io/pilotCfgStore` and `config.istio.io/accessConfigFile`
-annotations, although every other annotation and spec is validated for
-correct syntax.  They may be set to dummy values, as long as they are
-syntactically correct.
+1. Modify the name of the remote cluster's credentials file field
+`clusters.name` to the desired name of the cluster.
 
-If the prerequisites are met, the credentials for each Kubernetes cluster
-will also be present in `$HOME/multicluster`.
+1. Modify the name of the remote cluster's credential file field
+`contexts.context.cluster` to match the cluster name.
 
-Assemble and create the configmap from these files in the Kubernetes cluster
-where the Istio control plane is operational:
+## Instantiate the credentials for each remote cluster
+
+Execute this work on the cluster intended to run the Istio control
+plane.
+
+Create a namespace for instantiating the secrets:
+
+```bash
+kubectl create ns istio-system
+```
+
+Create a secret and label it properly for each remote cluster:
 
 ```bash
-kubectl create configmap -n istio-system multicluster -f $HOME/multicluster
+pushd $HOME/multicluster
+kubectl create secret generic ${CLUSTER_NAME} --from-file ${CLUSTER_NAME} -n istio-system
+kubectl label secret ${CLUSTER_NAME} istio/multiCluster=true -n istio-system
+popd
 ```
 
-## Deploy on the remote cluster
+<img src="{{home}}/img/exclamation-mark.svg" alt="Warning" title="Warning" style="width: 32px; display:inline" />Ordering currently matters.  Secrets must be created prior to the deployment of
+the Istio control plane.  Creating secrets after Istio is started will not register the
+secrets with Istio properly.
+
+## Deploy the Istio control plane
+
+Install the [Istio control plane]({{home}}/docs/setup/kubernetes/quick-start.html)
+on **one** Kubernetes cluster.
 
-Multicluster makes **one** Istio service mesh from all Kubernetes clusters
-that participate in multicluster.  Multicluster uses **one** Kubernetes
-cluster to run the Istio control plane.  **Any reasonable** number of remote
-Kubernetes clusters may be attached to the **one** Istio control plane.
+## Install the Istio remote on every remote
```

Comment:
s/Install the Istio remote on every remote/Install the Istio remote on every remote cluster